main.go

```diff
@@ -233,11 +293,12 @@ func main() {
 		log.Fatal("Must provide an API key!")
 	}
 
-	if shouldEncrypt == nil && shouldDecrypt == nil && shouldValidateCiphertext == nil {
+	if shouldEncrypt == nil && shouldDecrypt == nil && shouldValidateCiphertext == nil && shouldSign == nil && shouldVerify == nil {
 		log.Fatal("Must specify either encrypt OR decrypt OR is-peacemakr-ciphertext")
 	}
 
-	if shouldEncrypt != nil && shouldDecrypt != nil && shouldValidateCiphertext != nil && *shouldEncrypt && *shouldDecrypt && *shouldValidateCiphertext {
+	if shouldEncrypt != nil && shouldDecrypt != nil && shouldValidateCiphertext != nil && shouldSign != nil && shouldVerify != nil &&
+	   *shouldEncrypt && *shouldDecrypt && *shouldValidateCiphertext && *shouldSign && *shouldVerify {
 		log.Fatal("Must not attempt multiple functions simultaneously")
```

Comment:
I think the logic is checking if we try to use alllllll of the actions at once, not just more than 1 action?

Reply:
hmmm you are right. this doesn't  prevent client from running at least one action